src/olympia/amo/mail.py

```diff
@@ -17,30 +17,39 @@ class DevEmailBackend(BaseEmailBackend):
     `settings.SEND_REAL_EMAIL` is disabled.
 
     BUT even if `settings.SEND_REAL_EMAIL` is disabled, if the targeted
-    email address is in the `settings.EMAIL_QA_ALLOW_LIST` list,
-    the email will be sent.
+    email address is in `dev_email` group users, the email will be sent.
     """
 
+    @staticmethod
+    def force_send_mail_group():
+        return 'Force Send Mail'
```

Comment:
Probably best we keep the name descriptive and similar to the existing setting
```suggestion
        return 'QA Real Emails'
```
but @wagnerand because he'll be the one who has to remember what its for in the future 😄

Reply:
I voted force `Force Send Emails` or `Bypass Fake Email` in slack so that it would describe what the group does rather than who the group contains. I think it's better to do the former, it's not just about QA.

Reply:
I'd vote for "Bypass Fake Email" then, as it's the most descriptive.

Reply:
This basically wouldn't impact prod, since we don't use `FakeMail` there, correct? I guess we could even just not create that group on prod, if so.

Reply:
Ah, yes, you're right @wagnerand - it won't impact you.

Reply:
I'm not sure what I'm supposed to do at this point. Are we changing the name? Do we need to modify the migration script to check for prod/non-prod. I think that latter is more code complexity to remove a harmless outcome. The former seems to be a mix of opinions so unless someone can make a definitive decision I would leave as is as the easiest path forward.